ftw/slacker/slack_notifier.py

```diff
@@ -0,0 +1,87 @@
+from ftw.slacker.interfaces import ISlackNotifier
+from threading import Thread
+from zope.component import getUtility
+from zope.interface import implements
+import os
+import requests
+
+
+def notify_slack(*args, **kwargs):
+    """This is the main api mehtod to perform a slack notification
+
+    Always use this method to do slack notifications.
+
+    See the readme.rst to find out how to use it.
+    """
+    slacker = getUtility(ISlackNotifier)
+    return slacker.notify(*args, **kwargs)
+
+
+class SlackNotifier(object):
+    """The default slack notifier utility posts a message into slack through
+    a webhook.
+    """
+    implements(ISlackNotifier)
+
+    # Name of the request thread
+    THREAD_NAME = 'SlackNotifier-Thread'
+
+    # Use this string in your environment variables to deactivate
+    # notification.
+    NOTIFICATION_DEACTIVATION_VALUE = 'deactivate'
```

Comment:
I'd probably move this to module-level.
That also has the advantage that it can be imported from outside.